### PR TITLE
Add outgoing ip address to logs upon starting monitor

### PR DESCRIPTION
**Pull request by angjunkang**
This commit logs the outgoing IP address upon starting the monitor, eliminating the need for users to manually check their network configuration. This should improve ease of use, especially for non-technical users.

This can be seen below:

image

Do check and let me know if there is any issue. Thanks!

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -39,6 +40,10 @@ func init() {
 	}
 	restartThreshold = time.Duration(idleMinutes) * time.Minute
 	logDebug("idletime = " + restartThreshold.String())
+
+	ipAddress := GetOutboundIP().String()
```
Comment (mcs) [inline main.go]: You missed this one after making the method private:
```suggestion
	ipAddress := getOutboundIP().String()
```

Comment (angjunkang) [reply main.go]: Oh right! Sorry, I missed that out.

**Comment by mcs** [comment]
Could you also remove the `.vs` folder from the Pull Request? After that, I'll happily merge your change into the main branch 👍

**Comment by mcs** [comment]
OK, who cares. The folder won't hurt anyone. I'll merge it now. Thanks again for the contribution 😃

**Pull request merged**